src/frontend/context/TidslinjeContext.ts

```diff
@@ -118,23 +116,20 @@ const [TidslinjeProvider, useTidslinje] = createUseContext(() => {
             }, []);
     };
 
-    const mapPersonerTilPersonberegninger = (
+    const mapPersonerTilAndelerTilkjentYtelse = (
         personer: IGrunnlagPerson[],
-        personberegninger?: IPersonBeregning[]
-    ): IPersonBeregning[] => {
-        if (!personberegninger) {
```

Comment:
Veldig fine endringer av variabelnavn 👏  På grunn av all refaktoreringa som følge av dette er det litt vanskelig å trekke ut de andre endringene. 

💰 Hva forsøker du å løse i denne PR'en
Veldig fint og nyttig at det står at det ikke er funksjonelle endringer :) Kunne også gjerne tenkt meg en setning om forbedringen. Er det sånn at man tidligere i praksis genererte andeler tilkjent ytelse (personberegninger) fra beregningOversikt/IOppsummeringBeregning (nå utbetalingsperioder), men nå heller returnerer fra backend?

Reply:
Nei, ingen endringer der heller - bare navnendringer og forbedringer som jeg så underveis (type optional liste for eksempel)

Selve koden til tidlinjen har jeg ikke sett på.

Reply:
Men ser at koden til beregningsoversikt konsekvent er erstatta av disse to feltene? Andeler og utbetalingsperioder?

Reply:
Tja, personBeregninger=andelerTilkjentYtelse og beregningOversikt=utbetalingsperioder.

Reply:
Ja, jeg skjønner, men tidligere inneholdt inne IBehandling dette feltet. Derfor jeg lurer på om vi nå skal motta noe vi tidligere forsøkte å generere frontend?